data_layer/avail/avail.go

```diff
@@ -42,15 +44,78 @@ func (a *Avail) SetMetricsEndpoint(endpoint string) {
 }
 
 func NewAvail(root string) *Avail {
+	var daNetwork string
+
+	rollerData, err := roller.LoadConfig(root)
+	errorhandling.PrettifyErrorIfExists(err)
+
 	cfgPath := GetCfgFilePath(root)
 	availConfig, err := loadConfigFromTOML(cfgPath)
+
 	if err != nil {
-		entropySeed, err := bip39.NewEntropy(mnemonicEntropySize)
+		envs := []string{"mainnet", "turing"}
```

Comment:
for mainnet it should use mainnet, for any other environment, it should use testnet, user shouldn't be able to initialize Avail on mainnet for a testnet rollapp